lib/modules/manager/pep621/processors/pdm.ts

```diff
@@ -187,6 +187,11 @@ function generateCMDs(updatedDeps: Upgrade[]): string[] {
         );
         break;
       }
+      case depTypes.buildSystemRequires: {
+        // build requirements are not locked in the lock files, no need to update.
+        // Reference: https://github.com/pdm-project/pdm/discussions/2869
+        break;
+      }
```

Comment:
```suggestion
      case depTypes.buildSystemRequires:
        // build requirements are not locked in the lock files, no need to update.
        // Reference: https://github.com/pdm-project/pdm/discussions/2869
        break;
```
braces are not necessary here